node/src/GlideClient.ts

```diff
@@ -246,12 +247,12 @@ export class GlideClient extends BaseClient {
      * console.log(result); // Output: 'Hello'
      * ```
      */
-    public async ping(options?: {
-        message?: GlideString;
-        decoder?: Decoder;
-    }): Promise<GlideString> {
-        return this.createWritePromise(createPing(options?.message), {
-            decoder: options?.decoder,
+    public async ping(message?: GlideString): Promise<GlideString> {
+        return this.createWritePromise(createPing(message), {
```

Comment:
Please revert - this is not the desired behaviour.  
The message returned may not be the same type as the message sent.